plugins/scaffolder-backend-module-gitlab/src/actions/gitlabMergeRequest.ts

```diff
@@ -20,25 +20,56 @@ import {
   SerializedFile,
   serializeDirectoryContents,
 } from '@backstage/plugin-scaffolder-node';
-import { Types } from '@gitbeaker/core';
+import { Gitlab, Types } from '@gitbeaker/core';
 import path from 'path';
 import { ScmIntegrationRegistry } from '@backstage/integration';
 import { InputError } from '@backstage/errors';
 import { resolveSafeChildPath } from '@backstage/backend-plugin-api';
 import { createGitlabApi } from './helpers';
 import { examples } from './gitlabMergeRequest.examples';
+import { createHash } from 'crypto';
+import { Logger } from 'winston';
```

Comment:
Can we change this to `LoggerService` instead of `winston` logger type directly, or is it a requirement because of #26933